docs/src/blockchain-development/access_control.md

```diff
@@ -15,3 +15,21 @@ The `msg_sender` function works as follows:
 - If the caller is a contract, then `Result::Ok(Sender)` is returned with the `ContractId` sender variant.
 - If the caller is external (i.e. from a script), then all coin input owners in the transaction are checked. If all owners are the same, then `Result::Ok(Sender)` is returned with the `Address` sender variant.
 - If the caller is external and coin input owners are different, then the caller cannot be determined and a `Result::Err(AuthError)` is returned.
+
+## Contract Ownership
+
+Many contracts require some form of ownership and storing of an `Identity` for access control. When storing an `Identity` where the `BASE_ASSET_ID` should not be allowed as a default(such as ownership), it is recommended to use an `Option<Identity>` instead.
```

Comment:
How about something like this? I think this puts more emphasis on the good solution and less emphasis and the bad solution:
```suggestion
Many contracts require some form of ownership for access control. To accomplish this, it is recommended that a storage variable of type `Option<Identity>` is used to keep track of the owner. This allows setting and revoking ownership using the variants `Some(..)` and `None` respectively. This is better, safer, and more readable than using the `Identity` type directly where revoking ownership has to be done using some default value such as `BASE_ASSET_ID` or otherwise.
```

Reply:
Yes much better